src/components/CivicLikerV3/Page.vue

```diff
@@ -1,50 +1,109 @@
 <template>
-  <div class="civic-dashboard">
-    <div class="w-full mx-auto max-w-phone phone:px-12 tablet:px-12">
-      <header class="flex items-center justify-between mb-24">
-        <div class="flex items-center">
+  <div class="flex flex-col items-center px-[12px] laptop:px-[24px]">
+    <section class="w-full max-w-[976px] mx-auto">
+      <div
+        :class="[
+          'flex',
+          'flex-col',
+          'items-stretch',
+          'justify-center',
+          'w-full',
+          'h-full',
+          'rounded-[32px]',
+          'overflow-hidden',
+          'border-[1px]',
+          'border-shade-gray',
+
+          'laptop:flex-row',
+          'laptop:flex-nowrap',
+        ]"
+      >
+        <div class="flex flex-grow basis-2/3">
+          <img
+            v-if="$i18n.locale === 'zh-Hant'"
+            src="~/assets/images/civic-v3/keyBanner-zh.png"
+            :alt="$t('og_description')"
+            class="object-contain"
+          />
           <img
-            src="~/assets/images/civic-liker-icon.png"
-            :alt="$t('civic_page_v3_title')"
-            style="height:64px; width:68px"
+            v-else
+            src="~/assets/images/civic-v3/keyBanner-en.png"
+            :alt="$t('og_description')"
+            class="object-contain"
           />
-          <span class="ml-16 text-like-green text-24">{{
-            $t('civic_page_v3_title')
-          }}</span>
         </div>
-        <Button
-          :title="$t('civicLiker.about')"
-          preset="secondary-outline"
-          :href="$t('civic_dashboard_v3_about_url')"
-          target="_blank"
-          rel="noopener"
+        <div
+          class="flex flex-col justify-center flex-grow basis-1/3 h-full bg-like-green px-[32px] py-[46px]"
         >
-          <template #prepend>
-            <AlertCircle class="w-16 ml-4" />
-          </template>
-          <template #append>
-            <ExternalLink class="w-16" />
-          </template>
-        </Button>
-      </header>
-      <CivicDashboardV3 class="pb-64" />
-    </div>
+          <Label
+            class="text-white font-600 mb-[4px]"
+            :text="$t('civic_dashboard_v3_banner_writing_nft')"
+          />
+          <Label
+            class="text-like-cyan font-200 mb-[18px]"
+            :text="$t('civic_dashboard_v3_banner_writing_nft_description')"
+          />
+          <Label
+            class="text-white font-600 mb-[4px]"
+            :text="$t('civic_dashboard_v3_banner_book_nft')"
+          />
+          <Label
+            class="text-like-cyan font-200 mb-[24px]"
+            :text="$t('civic_dashboard_v3_banner_book_nft_description')"
+          />
+          <ButtonV2
+            preset="secondary"
+            :to="localeLocation({ name: 'store' })"
+            :text="$t('civic_dashboard_v3_banner_button')"
```

Comment:
Can we also have a click event here for analytics?

Reply:
[📈 Add redirect event for /civic](https://github.com/likecoin/liker-land/pull/1493/commits/be7e064409127dd45963b31c7c9a94081af5cc5c)